Sample/Kitchen Sink/DemonstrationTools/UnitConverterViewModel.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  Shape+Extensions.swift
+//  flight-ui-ios
+//
+//  Created by Appivate 2023
+//
+
+import Foundation
+import Combine
+import FlightUI
+import SwiftUI
+
+ class UnitConverterViewModel: ObservableObject {
+
+    @Published var kgInputString: String = ""
+    @Published var lbsInputString: String = ""
+    @Published var emptyFields = false
+    @Published var inputValue: String = ""
+    @Published var inputUnit: LengthType = .feet
```

Comment:
Minor: Do we use `inputUnit` and `outputUnit`? and with the bound values below, I don't think we need them to be optional 👍